src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
 // write your code here
+const logo = document.querySelector('.logo');
+const div = document.createElement('div');
+const divs = document.createElement('div');
```

Comment:
You are creating two separate div elements (`div` and `divs`) for displaying messages. Consider using a single div and updating its content and class based on the promise outcome to avoid redundancy.